File: BeerExercice/Armel/Models/Beer/Ingredient/Additive.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WikiBeer.Models
{
    internal class Additive : Ingredient
    {
        public string Use { get; internal set; }

        public Additive(string name, string use): base(name)
        {
            Use = use;
        }
    }
}

File: WikiBeer/Web/src/models/ingredients/additive.ts
import { Ingredient } from "./ingredient";

export class Additive extends Ingredient{

  private _use: string = '';

  constructor(id: string, name: string, description: string, use: string)
  {
    super(id, name, description);
    this._use = use;
  }

  get use(): string {return this._use;}
  set use(value: string) {this._use = value;}
}
